src/OpenTelemetry.Exporter.Jaeger/JaegerExporterOptionsExtensions.cs

```diff
@@ -0,0 +1,28 @@
+// <copyright file="JaegerExporterOptionsExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Exporter;
+
+internal static class JaegerExporterOptionsExtensions
+{
+    public static JaegerExportProtocol? ToJaegerExportProtocol(this string protocol) =>
+        protocol.Trim() switch
```

Comment:
nit: Maybe do `protocol?.Trim()` here just to be defensive.

Reply:
```suggestion
        protocol?.Trim() switch
```